src/blueprints/Do.Blueprints.Service.Application/HttpClient/HttpClientExtensions.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Do.Architecture;
+using Do.HttpClient;
+
+namespace Do;
+
+public static class HttpClientExtensions
+{
+    public static void AddHttpClient(this List<ILayer> source) =>
+        source.Add(new HttpClientLayer());
+    public static void ConfigureHttpClients(this LayerConfigurator source, Action<List<HttpClientDescriptor>> configuration) =>
+        source.Configure(configuration);
```

Comment:
rest of the project have these extensions in a single line, make this single line